src/CMakeLists.txt

```diff
@@ -0,0 +1,86 @@
+cmake_minimum_required(VERSION 3.13.0)
+
+set(DATADIR "/usr/local/share/gnotime")
+set(GETTEXT_PACKAGE "gnotime-2.0")
+set(GNOMELOCALEDIR "/usr/local/share/locale")
+set(GNOME_ICONDIR "NONE/share/pixmaps")
+set(GTTDATADIR "/usr/local/share/gnotime")
+set(GTTGLADEDIR "/usr/local/share/gnotime")
+set(LIBDIR "/usr/local/lib/x86_64-linux-gnu")
+set(PACKAGE "gnotime")
+set(PREFIX "/usr/local")
+set(SYSCONFDIR "etc")
+set(VERSION "${PROJECT_VERSION}${GTT_VERSION_SUFFIX}")
```

Comment:
With CMake I set the project version to `3.0.0` instead of `2.99.0` but therefore append a `_dev` suffix indicating the development. I prefer this style because it communicates both the intended upcoming version as well as that it's not finished yet.